src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String model;
```

Comment:
Don't forget about access modifiers for your custom fields. It's better to make 'model' and 'enginePower' private and access them through getters and setters.